fix: github action version (#71)
kubeshop/testkube-docs: docs/articles/github-actions.mdx

To connect to the self-hosted instance, you need to have **kubectl** configured for accessing your Kubernetes cluster, and simply passing optional namespace,
 if Testkube is not deployed in the default **testkube** namespace. 

> You can also specify a particular Testkube CLI or API version (don't confuse it with the Helm chart one), if it's omitted, then latest version is used.

Reviewer: ```suggestion
You can also specify a particular Testkube CLI or API version (not to be confused with the Helm chart one). If the version is omitted, then the latest version is used.
```

> The list of [releases] (https://github.com/kubeshop/testkube/releases) can be found in Testkube Github repository.

Reviewer: ```suggestion
The list of [releases] (https://github.com/kubeshop/testkube/releases) can be found in the Testkube Github repository.
```
